src/form/common/default-validation-conditionals.ts

```diff
@@ -451,3 +451,11 @@ export const detailsDontExist = '!values.detailsExist'
 // primary address same as other primary
 export const primaryAddressSameAsOtherPrimaryAddress =
   'values.primaryAddressSameAsOtherPrimary'
+
```

Comment:
`default-validation-conditionals.ts` what should this file actually contain? don't know if I fully understand the relationship between custom-conditionals and this one tbh

Reply:
Probably when the expression needs a custom JS function that is not already available in the eval?

Reply:
Yeah! my understanding was exactly it. However it obviously creates some confusion when we want to add a new conditional as there is no explicit guidance about it (from a country implementer perspective) 
May be we could get rid of `default` naming here, as it has no extra significance and put a documentation in `custom-validations.ts` file to make things a bit more explicit.... 🤔

Reply:
Or I can move this to `custom-conditionals.ts` if it is more intuitive

Reply:
Lets note that... no need to fix within this ticket though I think. Let's discuss with team

Reply:
@tahmidrahman-dsi No need I think, as it's passed to core as `.js` and it's not needed for this case. But some clarification required which goes where though within our team/documentation